signing_service/src/tests/tests.py

```diff
@@ -0,0 +1,59 @@
+from unittest import TestCase
+import mock
+import sys
+
+from ..constants import SIGNING_SERVICE_DEFAULT_PORT
+from ..signing_service import _parse_arguments
+from ..signing_service import main
+
+
+class SigningServiceMainTestCase(TestCase):
+
+    def test_that_main_should_called_with_correct_address(self):
+        with self.assertRaises(SystemExit):
+            main('not_really_ip_or_host', 8000)
+
+    def test_that_main_should_exit_when_called_with_wrong_address(self):
```

Comment:
Hm... shouldn't it be the name for the test above? ;)

Reply:
Yes :)